src/DssSpell.t.base.sol

```diff
@@ -856,6 +856,18 @@ contract DssSpellTestBase is Config, DssTest {
             bytes32(type(uint256).max)
         );
 
+        // Initially this test assume that's we are using freshly deployed Cliiper contract without any past auctions
+        if (clipper.kicks() > 0) {
```

Comment:
Since this is in base tests, I would be looking into why and when this was added / removed.

As crafter I might also check that tests pass both with and without this code snippet - if you do that, you can be reasonably sure the coverage is OK without having to dig into exactly how it plays around with stuff.

Either way I don't see any immediate harm from this, this is just because it's a base tests change (which is relatively rare)

Reply:
I think the comment clearly describes the reason why the block was added: the added block resets `kicks` variable to 0 so the rest of the test can assume it's a freshly deployed clipper. Moreover, as pointed out in the [PR description above](https://github.com/makerdao/spells-goerli/pull/195#issue-1705362783), this comment isn't even written by me, but copied from the [current mainnet version of the test](https://github.com/makerdao/spells-mainnet/blob/99a1bf8df1d262f421d91749ae9a3df366737328/src/DssSpell.t.base.sol#L887C14-L1017). After this change, the code for them is identical

Reply:
Good catch! 👍🏻